sklearn/ensemble/gradient_boosting_cv.py

```diff
@@ -0,0 +1,804 @@
+        There is a trade-off between learning_rate and n_estimators. If a list
+        is given, all learning rates will be evaluated.
+
+    max_depth : integer, list of integers, optional (default=3)
+        maximum depth of the individual regression estimators. The maximum
+        depth limits the number of nodes in the tree. Tune this parameter
+        for best performance; the best value depends on the interaction
+        of the input variables.
+        Ignored if ``max_leaf_nodes`` is not None.
+        If a list is given, each item in the list will be evaluated.
+
+    min_samples_split : int, float, list of int or float, optional (default=2)
+        The minimum number of samples required to split an internal node:
+        - If int, then consider `min_samples_split` as the minimum number.
+        - If float, then `min_samples_split` is a percentage and
+          `ceil(min_samples_split * n_samples)` are the minimum
+          number of samples for each split.
+        If a list is given, each item in the list will be evaluated.
+
+    min_samples_leaf : int, float, list of int or floats, optional (default=1)
+        The minimum number of samples required to be at a leaf node:
+        - If int, then consider `min_samples_leaf` as the minimum number.
+        - If float, then `min_samples_leaf` is a percentage and
+          `ceil(min_samples_leaf * n_samples)` are the minimum
+          number of samples for each node.
+        If a list is given, each item in the list will be evaluated.
+
+    min_weight_fraction_leaf : float, list of floats, optional (default=0.)
+        The minimum weighted fraction of the input samples required to be at a
+        leaf node. If a list is given, each item in the list will be evaluated.
+
+    subsample : float, list of floats, optional (default=1.0)
+        The fraction of samples to be used for fitting the individual base
+        learners. If smaller than 1.0 this results in Stochastic Gradient
+        Boosting. `subsample` interacts with the parameter `n_estimators`.
+        Choosing `subsample < 1.0` leads to a reduction of variance
+        and an increase in bias. If a list is given, each item in the list
+        will be evaluated.
+
+    max_features : int, float, string or None, list, optional (default=None)
+        The number of features to consider when looking for the best split:
+        - If int, then consider `max_features` features at each split.
+        - If float, then `max_features` is a percentage and
+          `int(max_features * n_features)` features are considered at each
+          split.
+        - If "auto", then `max_features=sqrt(n_features)`.
+        - If "sqrt", then `max_features=sqrt(n_features)`.
+        - If "log2", then `max_features=log2(n_features)`.
+        - If None, then `max_features=n_features`.
+        Choosing `max_features < n_features` leads to a reduction of variance
```

Comment:
Just double check the number of backticks everywhere, and proof-read the docstrings ;)